docs/server/source/server-reference/configuration.md

```diff
@@ -175,3 +176,17 @@ export BIGCHAINDB_STATSD_RATE=0.01
 ```js
 "statsd": {"host": "localhost", "port": 8125, "rate": 0.01}
 ```
+
+## backlog_reassign_delay
+
+Specifies how long, in seconds, transactions can remain in the backlog before being reassigned.  Long-waiting transactions must be reassigned because the assigned node may no longer be responsive.  The default duration is 30 seconds.
```

Comment:
It seems the default recently got changed to 120 seconds. See:

https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/__init__.py